src/api/utils/helpers.py

```diff
@@ -20,6 +23,14 @@
     ['data', 'count', 'page', 'pages', 'previous_url', 'next_url']
 )
 
+SUCCESS_OPS_MESSAGE = '''
+Hi {{name}},\n
+There are {{count}} pending logged activities. \n
+Have a great week ahead.\n
+Regards,
+Andela Societies
```

Comment:
Nitpick: 
Regards,
The Andela Societies Team.

Reply:
@LarryWachira

Reply:
Noted.